src/main/scala/is/hail/expr/AST.scala

```diff
@@ -660,17 +660,24 @@ case class Apply(posn: Position, fn: String, args: Array[AST]) extends AST(posn,
       } yield ir.ApplyBinaryPrimOp(op, x, y, t)
     }
 
+  def tryIRConversion(agg: Option[String]): Option[IR] =
```

Comment:
this should be `private[this]` since it's not intended to be used by other classes.